planning/behavior_path_planner/src/scene_module/avoidance/avoidance_module.cpp

```diff
@@ -2052,6 +2052,10 @@ BehaviorModuleOutput AvoidanceModule::plan()
   }
   output.path = std::make_shared<PathWithLaneId>(avoidance_path.path);
 
+  output.modified_goal.header.frame_id = planner_data_->route_handler->getRouteHeader().frame_id;
+  output.modified_goal.header.stamp = clock_->now();
```

Comment:
think about which stamp is better.

Reply:
chaned to route stamp in 
https://github.com/autowarefoundation/autoware.universe/pull/2094/commits/809a125f05665f7c37171cb206ca22a00b505ca0

Reply:
tier4 interanal slack https://star4.slack.com/archives/C4P0NSMB5/p1666069602035659?thread_ts=1665625570.381429&cid=C4P0NSMB5